api/controllers/TestController.js

```diff
@@ -95,4 +95,21 @@ module.exports = {
       return res.serverError(err);
     }
   },
+
+  getSpectators: async function(_req, res) {
+    try {
+      const spectators = await UserSpectatingGame.find({})
+        .populate('spectator');
+      const result = spectators.map(spectatingRecord => {
+        return {
+          ...spectatingRecord,
+          spectator: spectatingRecord.spectator.username
+        };
+      });
+
+      return res.json(result);
+    } catch (err) {
+      return res.serverError(err);
+    }
+  }
```

Comment:
Added this to make it easier to test whether spectators are inactivated when they leave (this was failing because `stores/game.js` had the wrong url for the spectate/leave endpoint). In addition to fixing the behavior, we should test that it works. This way we can get all the spectators, assert how many there are, and that they have the correct date